packages/analytics-js-common/src/types/ApplicationState.ts

```diff
@@ -83,6 +83,10 @@ export type LifecycleState = {
   dataPlaneUrl: Signal<string | undefined>;
 };
 
+export type AutoTrackState = {
```

Comment:
Let's add the `enabled` status of auto tracking and page life cycle to the state.

Reply:
Addressed